source/reference/json.txt

```diff
@@ -1,8 +1,106 @@
+.. _json-cluster-config:
 
 ======
 |json|
 ======
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The {+atlas-cli+} accepts ``.json`` configuration files. You can use a 
+configuration file to specify settings when using the {+atlas-cli+} to:
+
+- :ref:`Create <atlas-clusters-create>`, :ref:`update <atlas-clusters-update>`, 
+  or :ref:`upgrade a cluster <atlas-clusters-upgrade>`.
+- :ref:`Update a cloud backup schedule <atlas-backups-schedule-update>`.
+- :ref:`Create <atlas-clusters-search-indexes-create>` or 
+  :ref:`update an Atlas Search Index <atlas-clusters-search-indexes-update>`.
+
+Usage
+-----
+
+To learn which settings are accepted by the {+atlas-cli+}, view the request 
+body schema and copy the request samples in the :ref:`Atlas Admin API Specification 
+<api-resources-spec>` for the endpoint that corresponds to the {+atlas-cli+} 
+command:
+
+.. list-table::
```

Comment:
[q/not blocking] This looks like a potential maintenance issue - if new commands are added with `--file` I could see it being easy to forget to update this page.

Should we aim for a more generic text here?

Reply:
Really great point - and thanks for bringing this up! I talked over Slack with @sarahsimpers and she's opening a ticket to think about ways to maintain this page (e.g an automated ticket generated if a file contains --file). For now, I've revised the text to be slightly more general/clarify that other commands might accept the `--file` option as well.